src/app/core/services/notifications/fcm-rest-notification.service.ts

```diff
@@ -149,21 +147,20 @@ export class FcmRestNotificationService extends FcmNotificationService {
       })
   }
 
-  cancelAllNotifications(subject): Promise<any> {
-    return this.appServerService
-      .pullAllPublishedNotifications(subject)
-      .then((res: FcmNotifications) => {
-        const now = Date.now()
-        const notifications = res.notifications
-          .map(n => ({
-            id: n.id,
-            timestamp: getMilliseconds({ seconds: n.scheduledTime })
-          }))
-          .filter(n => n.timestamp > now)
-        notifications.map(o => this.cancelSingleNotification(subject, o))
+  sendNotificationsBundle(notifications, subjectId, projectId) {
+    return this.appServerService.addNotificationsBundle(notifications, subjectId, projectId)
+      .then((resultNotification: FcmNotificationDto) => {
+        this.setLastNotificationUpdate(Date.now())
+        // notification.notification.id = resultNotification.id
```

Comment:
Would it be possible to still store back the notification id? We would still need it when we want to cancel reminders/future notifications for a task once it has been completed.

Reply:
That would be done bij uncommenting the lines below right?

Reply:
No cause I think the request would return multiple notifications and each would be needed to get stored back to the task object.